### ISSUE TITLE
Is my bootVm supposed to look like this 

### ISSUE THREAD

**Issue by stylesp354** [ra1nstorm/ra1nstorm-helper#39]
![64FC0929-40C9-449B-A12F-FBC8ACB30933](https://user-images.githubusercontent.com/58155886/69569419-9ef5c880-0f8b-11ea-96b8-3a8aa6869210.jpeg)

**Comment by ObviouslyAce** [comment]
yes, that's how it's supposed to look like. If you don't know how to open it just rerun ra1nstorm and click boot VM and it'll boot ur VM up.

**Comment by stylesp354** [comment]
When I click boot.vm nothing happens it says I don’t have permissions

**Comment by ObviouslyAce** [comment]
r u clicking the file or...

**Comment by ObviouslyAce** [comment]
when u run ra1nstorm do this "sudo bash *drop file of ra1nstorm here*"

**Comment by stylesp354** [comment]
So basically I type in “sudo bash” and then slide the file into terminal ?

**Comment by ObviouslyAce** [comment]
Yes
